examples/atari/atari_iqn.py

```diff
@@ -0,0 +1,183 @@
+import os
+import torch
+import pprint
+import argparse
+import numpy as np
+from torch.utils.tensorboard import SummaryWriter
+
+from tianshou.utils import BasicLogger
+from tianshou.utils.net.discrete import ImplicitQuantileNetwork
+from tianshou.policy import IQNPolicy
+from tianshou.env import SubprocVectorEnv
+from tianshou.trainer import offpolicy_trainer
+from tianshou.data import Collector, VectorReplayBuffer
+
+from atari_network import DQN
+from atari_wrapper import wrap_deepmind
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--task', type=str, default='PongNoFrameskip-v4')
+    parser.add_argument('--seed', type=int, default=1234)
+    parser.add_argument('--eps-test', type=float, default=0.005)
+    parser.add_argument('--eps-train', type=float, default=1.)
+    parser.add_argument('--eps-train-final', type=float, default=0.05)
+    parser.add_argument('--buffer-size', type=int, default=100000)
+    parser.add_argument('--lr', type=float, default=0.0001)
+    parser.add_argument('--gamma', type=float, default=0.99)
+    parser.add_argument('--sample-size', type=int, default=32)
+    parser.add_argument('--online-sample-size', type=int, default=8)
+    parser.add_argument('--target-sample-size', type=int, default=8)
+    parser.add_argument('--num-cosines', type=int, default=64)
+    parser.add_argument('--hidden-sizes', type=int, nargs='*', default=[512, 512])
```

Comment:
why did you change this?

Reply:
This is to make the network structure comparable to QRDQN. This `hidden-sizes` parameter controls the MLP structure **after** the feature net. See https://github.com/thu-ml/tianshou/blob/458028a326fed7bffacd42b457a9a55e933a7bd3/examples/atari/atari_network.py#L34-L37

Reply:
Yes but this equals to `[512]` instead of `[512, 512]`.

Reply:
Ah, you are right. I'll change it back and rerun the experiments.